src/qt/utilitydialog.h

```diff
@@ -24,7 +24,7 @@ class HelpMessageDialog : public QDialog
     Q_OBJECT
 
 public:
-    explicit HelpMessageDialog(interfaces::Node& node, QWidget *parent, bool about);
+    explicit HelpMessageDialog(interfaces::Node& node, QWidget *parent, bool about, bool specHelp = false);
```

Comment:
nit: `spec` might also be read as `specific` or `specification` when a new reader comes into the codebase. I think naming it with the full `specialHelp` you use in https://github.com/bitcoin/bitcoin/pull/17901/files#diff-8c9d79ba40bf702f01685008550ac100R482 would make it easier on new readers to understand what's going on.